xml/options-3.4.rng

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<grammar xmlns="http://relaxng.org/ns/structure/1.0"
+         datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
+  <start>
+    <ref name="options"/>
+  </start>
+
+  <!--
+   see upgrade-2.10.xsl
+   - cibtr:table for="cluster-properties"
+   -->
+  <define name="cluster_property_set.nvpair.name-value-unsupported">
```

Comment:
@jnpkrn , I forget whether we need to keep these "unsupported" blocks once the relevant file moves beyond 3.0 -- were they only needed for the 3.0 schema specifically, or do we need to keep them for all 3.x?